WcaOnRails/app/models/registration.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 class Registration < ActiveRecord::Base
-  scope :pending, -> { where(accepted_at: nil) }
-  scope :accepted, -> { where.not(accepted_at: nil) }
+  scope :pending, -> { where(accepted_at: nil).where(deleted_at: nil) }
+  scope :accepted, -> { where.not(accepted_at: nil).where(deleted_at: nil) }
+  scope :deleted, -> { where.not(deleted_at: nil) }
```

Comment:
It looks like you opted not to go with @larspetrus's suggestion to use `default_scope` here so people will not have to "just remember" that this model gets soft deleted? Just checking that this was intentional.